new module: damageprofiler

https://github.com/Integrative-Transcriptomics/DamageProfiler

Typical usage:
```
damageprofiler -i bam_file -r reference_fasta -o output_directory
```

Already used in nf-core pipelines such as [Eager](https://nf-co.re/eager), and [coproID](https://nf-co.re/coproid)

Issue of running a bioconda package with biocontainer (docker container).
The (java) package works fine when using it with conda, but fails with
```
Caused by: java.lang.NullPointerException
    at java.desktop/sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1262)
```
when used through docker.

Looks like the solution is updating the conda package with `libfontconfig1` https://github.com/AdoptOpenJDK/openjdk-docker/issues/75

The bioconda package is https://bioconda.github.io/recipes/damageprofiler/README.html?highlight=damageprofiler#package-package%20'damageprofiler&#x27;

--- a/software/damageprofiler/functions.nf
+++ b/software/damageprofiler/functions.nf
@@ -1,0 +1,68 @@
+//
+//  Utility functions used in nf-core DSL2 module files
+//
+
+//
+// Extract name of software tool from process name using $task.process
+//
+def getSoftwareName(task_process) {
+    return task_process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()
+}
+
+//
+// Function to initialise default values and to generate a Groovy Map of available options for nf-core modules
+//
+def initOptions(Map args) {
+    def Map options = [:]
+    options.args            = args.args ?: ''
+    options.args2           = args.args2 ?: ''
+    options.args3           = args.args3 ?: ''
+    options.publish_by_meta = args.publish_by_meta ?: []
+    options.publish_dir     = args.publish_dir ?: ''
+    options.publish_files   = args.publish_files
+    options.suffix          = args.suffix ?: ''
+    return options
+}
+
+//
+// Tidy up and join elements of a list to return a path string
+//
+def getPathFromList(path_list) {
+    def paths = path_list.findAll { item -> !item?.trim().isEmpty() }      // Remove empty entries
+    paths     = paths.collect { it.trim().replaceAll("^[/]+|[/]+\$", "") } // Trim whitespace and trailing slashes
+    return paths.join('/')
+}
+
+//
+// Function to save/publish module results
+//
+def saveFiles(Map args) {
+    if (!args.filename.endsWith('.version.txt')) {
+        def ioptions  = initOptions(args.options)
+        def path_list = [ ioptions.publish_dir ?: args.publish_dir ]
+        if (ioptions.publish_by_meta) {
+            def key_list = ioptions.publish_by_meta instanceof List ? ioptions.publish_by_meta : args.publish_by_meta
+            for (key in key_list) {
+                if (args.meta && key instanceof String) {
+                    def path = key
+                    if (args.meta.containsKey(key)) {
+                        path = args.meta[key] instanceof Boolean ? "${key}_${args.meta[key]}".toString() : args.meta[key]
+                    }
+                    path = path instanceof String ? path : ''
+                    path_list.add(path)
+                }
+            }
+        }
+        if (ioptions.publish_files instanceof Map) {
+            for (ext in ioptions.publish_files) {
+                if (args.filename.endsWith(ext.key)) {
+                    def ext_list = path_list.collect()
+                    ext_list.add(ext.value)
+                    return "${getPathFromList(ext_list)}/$args.filename"
+                }
+            }
+        } else if (ioptions.publish_files == null) {
+            return "${getPathFromList(path_list)}/$args.filename"
+        }
+    }
+}

--- a/software/damageprofiler/main.nf
+++ b/software/damageprofiler/main.nf
@@ -11,15 +11,11 @@ process DAMAGEPROFILER {
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
 
-    // TODO nf-core: List required Conda package(s).
-    //               Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
-    //               For Conda, the build (i.e. "h9402c20_2") must be EXCLUDED to support installation on different operating systems.
-    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
     conda (params.enable_conda ? "bioconda::damageprofiler=1.1" : null)
     if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/YOUR-TOOL-HERE"
+        container "https://depot.galaxyproject.org/singularity/damageprofiler:1.1--hdfd78af_1"
     } else {
-        container "quay.io/biocontainers/YOUR-TOOL-HERE"
+        container "quay.io/biocontainers/damageprofiler:1.1--hdfd78af_1"
     }
 
     input:

--- a/software/damageprofiler/main.nf
+++ b/software/damageprofiler/main.nf
@@ -13,9 +13,9 @@ process DAMAGEPROFILER {
 
     conda (params.enable_conda ? "bioconda::damageprofiler=1.1" : null)
     if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/damageprofiler:1.1--hdfd78af_1"
+        container "https://depot.galaxyproject.org/singularity/damageprofiler:1.1--hdfd78af_2"
     } else {
-        container "quay.io/biocontainers/damageprofiler:1.1--hdfd78af_1"
+        container "quay.io/biocontainers/damageprofiler:1.1--hdfd78af_2"
     }
 
     input:

--- a/software/damageprofiler/main.nf
+++ b/software/damageprofiler/main.nf
@@ -1,0 +1,48 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process DAMAGEPROFILER {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    // TODO nf-core: List required Conda package(s).
+    //               Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
+    //               For Conda, the build (i.e. "h9402c20_2") must be EXCLUDED to support installation on different operating systems.
+    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
+    conda (params.enable_conda ? "bioconda::damageprofiler=1.1" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/YOUR-TOOL-HERE"
+    } else {
+        container "quay.io/biocontainers/YOUR-TOOL-HERE"
+    }
+
+    input:
+    tuple val(meta), path(bam)
+    path fasta
+    path fai
+
+    output:
+    tuple val(meta), path("${prefix}"), emit: results
+    path  "*.version.txt"             , emit: version
+
+    script:
+    def software   = getSoftwareName(task.process)
+    prefix         = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+
+    """
+    damageprofiler \\
+        -i $bam \\
+        -r $fasta \\
+        -o $prefix/ \\
+        $options.args
+
+
+    echo \$(damageprofiler -v) | sed 's/^DamageProfiler v//' > ${software}.version.txt
+    """
+}

--- a/software/damageprofiler/meta.yml
+++ b/software/damageprofiler/meta.yml
@@ -1,0 +1,59 @@
+name: damageprofiler
+## TODO nf-core: Add a description of the module and list keywords
+description: A Java based tool to determine damage patterns on ancient DNA as a replacement for mapDamage
+keywords:
+  - damage
+  - deamination
+  - miscoding lesions
+  - C to T
+  - ancient DNA
+  - aDNA
+  - palaeogenomics
+  - paleogenomics
+  - palaeogenetics
+  - paleogenetics
+  - archaeogenetics
+  - archeogenetics
+  - archaeogenomics
+  - archeogenomics
+tools:
+  - damageprofiler:
+      ## TODO nf-core: Add a description and other details for the software below
+      description: A Java based tool to determine damage patterns on ancient DNA as a replacement for mapDamage
+      homepage: https://github.com/Integrative-Transcriptomics/DamageProfiler
+      documentation: https://damageprofiler.readthedocs.io/
+      tool_dev_url: https://github.com/Integrative-Transcriptomics/DamageProfiler
+      doi: "10.1093/bioinformatics/btab190"
+      licence: ['GPL v3']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+  - fasta:
+      type: file
+      description: FASTA reference file
+      pattern: "*.{fasta,fna,fa}"
+  - fai:
+      type: file
+      description: FASTA index file from samtools faidx
+      pattern: "*.{fai}"
+
+output:
+  - version:
+      type: file
+      description: File containing software version
+      pattern: "*.{version.txt}"
+  - results:
+      type: dir
+      description: DamageProfiler results directory
+      pattern: "*/*"
+
+authors:
+  - "@jfy133"

--- a/software/damageprofiler/meta.yml
+++ b/software/damageprofiler/meta.yml
@@ -8,13 +8,9 @@ keywords:
   - ancient DNA
   - aDNA
   - palaeogenomics
-  - paleogenomics
-  - palaeogenetics
-  - paleogenetics
-  - archaeogenetics
-  - archeogenetics
   - archaeogenomics
-  - archeogenomics
+  - palaeogenetics
+  - archaeogenetics
 tools:
   - damageprofiler:
       description: A Java based tool to determine damage patterns on ancient DNA as a replacement for mapDamage

--- a/software/damageprofiler/meta.yml
+++ b/software/damageprofiler/meta.yml
@@ -1,5 +1,4 @@
 name: damageprofiler
-## TODO nf-core: Add a description of the module and list keywords
 description: A Java based tool to determine damage patterns on ancient DNA as a replacement for mapDamage
 keywords:
   - damage
@@ -18,7 +17,6 @@ keywords:
   - archeogenomics
 tools:
   - damageprofiler:
-      ## TODO nf-core: Add a description and other details for the software below
       description: A Java based tool to determine damage patterns on ancient DNA as a replacement for mapDamage
       homepage: https://github.com/Integrative-Transcriptomics/DamageProfiler
       documentation: https://damageprofiler.readthedocs.io/

--- a/tests/config/pytest_software.yml
+++ b/tests/config/pytest_software.yml
@@ -190,6 +190,10 @@ cutadapt:
   - software/cutadapt/**
   - tests/software/cutadapt/**
 
+damageprofiler:
+  - software/damageprofiler/**
+  - tests/software/damageprofiler/**
+
 deeptools/computematrix:
   - software/deeptools/computematrix/**
   - tests/software/deeptools/computematrix/**

--- a/tests/software/damageprofiler/main.nf
+++ b/tests/software/damageprofiler/main.nf
@@ -7,9 +7,17 @@ include { DAMAGEPROFILER } from '../../../software/damageprofiler/main.nf' addPa
 workflow test_damageprofiler {
 
     input = [ [ id:'test', single_end:false ], // meta map
-              file(params.test_data['homo_sapiens']['illumina']['test.paired_end.sorted.bam'], checkIfExists: true) ]
-    fasta = file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
-    fai = file(params.test_data['homo_sapiens']['genome']['genome_fasta.fai'], checkIfExists: true)
+              [ file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true) ] ]
 
-    DAMAGEPROFILER ( input )
+    print(input)
+
+    fasta = file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
+
+    print(fasta)
+
+    fai = file(params.test_data['homo_sapiens']['genome']['genome_fasta_fai'], checkIfExists: true)
+
+    print(fai)
+
+    DAMAGEPROFILER ( input, fasta, fai )
 }

--- a/tests/software/damageprofiler/main.nf
+++ b/tests/software/damageprofiler/main.nf
@@ -8,16 +8,8 @@ workflow test_damageprofiler {
 
     input = [ [ id:'test', single_end:false ], // meta map
               [ file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true) ] ]
-
-    print(input)
-
     fasta = file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
-
-    print(fasta)
-
     fai = file(params.test_data['homo_sapiens']['genome']['genome_fasta_fai'], checkIfExists: true)
-
-    print(fai)
 
     DAMAGEPROFILER ( input, fasta, fai )
 }

--- a/tests/software/damageprofiler/main.nf
+++ b/tests/software/damageprofiler/main.nf
@@ -1,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { DAMAGEPROFILER } from '../../../software/damageprofiler/main.nf' addParams( options: [:] )
+
+workflow test_damageprofiler {
+
+    input = [ [ id:'test', single_end:false ], // meta map
+              file(params.test_data['homo_sapiens']['illumina']['test.paired_end.sorted.bam'], checkIfExists: true) ]
+    fasta = file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
+    fai = file(params.test_data['homo_sapiens']['genome']['genome_fasta.fai'], checkIfExists: true)
+
+    DAMAGEPROFILER ( input )
+}

--- a/tests/software/damageprofiler/test.yml
+++ b/tests/software/damageprofiler/test.yml
@@ -32,8 +32,6 @@
       should_exist: true
     - path: output/damageprofiler/test/editDistance.txt
       md5sum: af2d2f4a99058ec56eae88ec27779e38
-      #contains:
-      #  - "#Edit distances for file:test.paired_end.sorted.bam"
     - path: output/damageprofiler/test/Length_plot_combined_data.svg
       should_exist: true
     - path: output/damageprofiler/test/Length_plot_forward_reverse_separated.svg

--- a/tests/software/damageprofiler/test.yml
+++ b/tests/software/damageprofiler/test.yml
@@ -1,63 +1,46 @@
 - name: damageprofiler
-  command: nextflow run ./tests/software/damageprofiler -entry test_damageprofiler -c tests/config/nextflow.config
+  command: nextflow run ./tests/software/damageprofiler -entry test_damageprofiler -c tests/config/nextflow.config -dump-channels
   tags:
     - damageprofiler
   files:
-    - path: output/damageprofiler/test.paired_end/3p_freq_misincorporations.txt
-      md5sum: 37cbc08e075174b3b1203b0b6f2c8478
-    - path: output/damageprofiler/test.paired_end/3pGtoA_freq.txt
-      md5sum: cf193f50b349c3d376d65f5d3b363f35
-    - path: output/damageprofiler/test.paired_end/5pCtoT_freq.txt
-      md5sum: 5ee5bd71e2c45467cdeb6c2327c153ba
-    - path: output/damageprofiler/test.paired_end/5p_freq_misincorporations.txt
-      md5sum: 680c87a4e33361f6a4652b6317de72fc
-    - path: output/damageprofiler/test.paired_end/DamagePlot_five_prime.svg
-      md5sum: b99dc7079c38b830bf036abb93e1ecbd
-    - path: output/damageprofiler/test.paired_end/DamagePlot.pdf
-      md5sum: dabddf18f339d6d8877f9f05f8b354a8
-    - path: output/damageprofiler/test.paired_end/DamagePlot_three_prime.svg
-      md5sum: 3aa1b3cc8b1ae8e0365dcfaf9f6e03de
-    - path: output/damageprofiler/test.paired_end/DamageProfiler.log
-      md5sum: 7432ff4ab7ad061ca4d1c97ac6bc6f50
-    - path: output/damageprofiler/test.paired_end/dmgprof.json
+    - path: output/damageprofiler/test/3p_freq_misincorporations.txt
+      md5sum: da4cac90c78899a7cb6d72d415392b49
+    - path: output/damageprofiler/test/3pGtoA_freq.txt
+      md5sum: 8dab75d51a4b943b501d0995169c767f
+    - path: output/damageprofiler/test/5pCtoT_freq.txt
+      md5sum: fcc48ee5f72edff930d627c8bfdd8a5b
+    - path: output/damageprofiler/test/5p_freq_misincorporations.txt
+      md5sum: 54665474f5ef17dcc268567e5eaa7d86
+    - path: output/damageprofiler/test/DamagePlot_five_prime.svg
+      should_exist: true
+    - path: output/damageprofiler/test/DamagePlot.pdf
+      should_exist: true
+    - path: output/damageprofiler/test/DamagePlot_three_prime.svg
+      should_exist: true
+    - path: output/damageprofiler/test/DamageProfiler.log
+      contains:
+        - "FINISHED SUCCESSFULLY"
+    - path: output/damageprofiler/test/dmgprof.json
       md5sum: 98499024c7e937896e481f2d3cfbdd3e
-    - path: output/damageprofiler/test.paired_end/DNA_comp_genome.txt
-      md5sum: b22e01d1be28e0cf80af3197d83be2c7
-    - path: output/damageprofiler/test.paired_end/DNA_composition_sample.txt
-      md5sum: 46e2fe273af8e9fd7a4b442e5380cc8c
-    - path: output/damageprofiler/test.paired_end/edit_distance.pdf
-      md5sum: ea9d10e710d9c2e16e832adad4b8b0ef
-    - path: output/damageprofiler/test.paired_end/edit_distance.svg
-      md5sum: b61486f31d579191cd88c8cd7615315d
-    - path: output/damageprofiler/test.paired_end/editDistance.txt
-      md5sum: 233b27a7dd4ddef87d4470818c1f79c4
-    - path: output/damageprofiler/test.paired_end/Length_plot_combined_data.svg
-      md5sum: d1e1363be3bd6483076d1113b323a7e5
-    - path: output/damageprofiler/test.paired_end/Length_plot_forward_reverse_separated.svg
-      md5sum: 9157c7145daf88f90289f643c0de7dd1
-    - path: output/damageprofiler/test.paired_end/Length_plot.pdf
-      md5sum: dc89716f5fd82bd028f967cb5ecf8c21
-    - path: output/damageprofiler/test.paired_end/lgdistribution.txt
-      md5sum: d924bc3f73e95a7359c5675d15055713
-    - path: output/damageprofiler/test.paired_end/misincorporation.txt
-      md5sum: 91807be7419430c9fa7611134fc63cb5
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+    - path: output/damageprofiler/test/DNA_comp_genome.txt
+      md5sum: f91e70760d91a1193a27e360aaddf2fd
+    - path: output/damageprofiler/test/DNA_composition_sample.txt
+      md5sum: 1257eb3eb42484647bfba2151f9ef04f
+    - path: output/damageprofiler/test/edit_distance.pdf
+      should_exist: true
+    - path: output/damageprofiler/test/edit_distance.svg
+      should_exist: true
+    - path: output/damageprofiler/test/editDistance.txt
+      md5sum: af2d2f4a99058ec56eae88ec27779e38
+      #contains:
+      #  - "#Edit distances for file:test.paired_end.sorted.bam"
+    - path: output/damageprofiler/test/Length_plot_combined_data.svg
+      should_exist: true
+    - path: output/damageprofiler/test/Length_plot_forward_reverse_separated.svg
+      should_exist: true
+    - path: output/damageprofiler/test/Length_plot.pdf
+      should_exist: true
+    - path: output/damageprofiler/test/lgdistribution.txt
+      md5sum: c5d029bf3a92b613310ee23f47d94981
+    - path: output/damageprofiler/test/misincorporation.txt
+      md5sum: 3aa6dd749010a492d92a815a83c196a8

--- a/tests/software/damageprofiler/test.yml
+++ b/tests/software/damageprofiler/test.yml
@@ -12,11 +12,8 @@
     - path: output/damageprofiler/test/5p_freq_misincorporations.txt
       md5sum: 54665474f5ef17dcc268567e5eaa7d86
     - path: output/damageprofiler/test/DamagePlot_five_prime.svg
-      should_exist: true
     - path: output/damageprofiler/test/DamagePlot.pdf
-      should_exist: true
     - path: output/damageprofiler/test/DamagePlot_three_prime.svg
-      should_exist: true
     - path: output/damageprofiler/test/DamageProfiler.log
       contains:
         - "FINISHED SUCCESSFULLY"
@@ -27,17 +24,12 @@
     - path: output/damageprofiler/test/DNA_composition_sample.txt
       md5sum: 1257eb3eb42484647bfba2151f9ef04f
     - path: output/damageprofiler/test/edit_distance.pdf
-      should_exist: true
     - path: output/damageprofiler/test/edit_distance.svg
-      should_exist: true
     - path: output/damageprofiler/test/editDistance.txt
       md5sum: af2d2f4a99058ec56eae88ec27779e38
     - path: output/damageprofiler/test/Length_plot_combined_data.svg
-      should_exist: true
     - path: output/damageprofiler/test/Length_plot_forward_reverse_separated.svg
-      should_exist: true
     - path: output/damageprofiler/test/Length_plot.pdf
-      should_exist: true
     - path: output/damageprofiler/test/lgdistribution.txt
       md5sum: c5d029bf3a92b613310ee23f47d94981
     - path: output/damageprofiler/test/misincorporation.txt

--- a/tests/software/damageprofiler/test.yml
+++ b/tests/software/damageprofiler/test.yml
@@ -1,0 +1,63 @@
+- name: damageprofiler
+  command: nextflow run ./tests/software/damageprofiler -entry test_damageprofiler -c tests/config/nextflow.config
+  tags:
+    - damageprofiler
+  files:
+    - path: output/damageprofiler/test.paired_end/3p_freq_misincorporations.txt
+      md5sum: 37cbc08e075174b3b1203b0b6f2c8478
+    - path: output/damageprofiler/test.paired_end/3pGtoA_freq.txt
+      md5sum: cf193f50b349c3d376d65f5d3b363f35
+    - path: output/damageprofiler/test.paired_end/5pCtoT_freq.txt
+      md5sum: 5ee5bd71e2c45467cdeb6c2327c153ba
+    - path: output/damageprofiler/test.paired_end/5p_freq_misincorporations.txt
+      md5sum: 680c87a4e33361f6a4652b6317de72fc
+    - path: output/damageprofiler/test.paired_end/DamagePlot_five_prime.svg
+      md5sum: b99dc7079c38b830bf036abb93e1ecbd
+    - path: output/damageprofiler/test.paired_end/DamagePlot.pdf
+      md5sum: dabddf18f339d6d8877f9f05f8b354a8
+    - path: output/damageprofiler/test.paired_end/DamagePlot_three_prime.svg
+      md5sum: 3aa1b3cc8b1ae8e0365dcfaf9f6e03de
+    - path: output/damageprofiler/test.paired_end/DamageProfiler.log
+      md5sum: 7432ff4ab7ad061ca4d1c97ac6bc6f50
+    - path: output/damageprofiler/test.paired_end/dmgprof.json
+      md5sum: 98499024c7e937896e481f2d3cfbdd3e
+    - path: output/damageprofiler/test.paired_end/DNA_comp_genome.txt
+      md5sum: b22e01d1be28e0cf80af3197d83be2c7
+    - path: output/damageprofiler/test.paired_end/DNA_composition_sample.txt
+      md5sum: 46e2fe273af8e9fd7a4b442e5380cc8c
+    - path: output/damageprofiler/test.paired_end/edit_distance.pdf
+      md5sum: ea9d10e710d9c2e16e832adad4b8b0ef
+    - path: output/damageprofiler/test.paired_end/edit_distance.svg
+      md5sum: b61486f31d579191cd88c8cd7615315d
+    - path: output/damageprofiler/test.paired_end/editDistance.txt
+      md5sum: 233b27a7dd4ddef87d4470818c1f79c4
+    - path: output/damageprofiler/test.paired_end/Length_plot_combined_data.svg
+      md5sum: d1e1363be3bd6483076d1113b323a7e5
+    - path: output/damageprofiler/test.paired_end/Length_plot_forward_reverse_separated.svg
+      md5sum: 9157c7145daf88f90289f643c0de7dd1
+    - path: output/damageprofiler/test.paired_end/Length_plot.pdf
+      md5sum: dc89716f5fd82bd028f967cb5ecf8c21
+    - path: output/damageprofiler/test.paired_end/lgdistribution.txt
+      md5sum: d924bc3f73e95a7359c5675d15055713
+    - path: output/damageprofiler/test.paired_end/misincorporation.txt
+      md5sum: 91807be7419430c9fa7611134fc63cb5
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+

